server/src/test/ZetaSqlWrapper.spec.ts

```diff
@@ -35,55 +72,86 @@ describe('ZetaSqlWrapperTest', () => {
-    assert.ok(tables);
-    assertThat(tables.size, 1);
-    assertThat(tables.get(table)?.name, table);
+    const dataSetCatalog = datasets?.get(expectedDataSet);
+    assert.ok(dataSetCatalog);
+    assertThat(dataSetCatalog.catalogs.size, 1);
 
-    const columns = tables.get(table)?.columns;
-    assert.ok(columns);
-    assertThat(columns.length, expectedColumns.length);
-    assertThat(columns.map((c: SimpleColumn) => c.getName()).sort(), expectedColumns.sort());
+    const informationSchemaCatalog = dataSetCatalog.catalogs.get(InformationSchemaConfigurator.INFORMATION_SCHEMA);
+    assert.ok(informationSchemaCatalog);
+    assertThat(informationSchemaCatalog.tables.size, 1);
+
+    const table = informationSchemaCatalog.tables.get(expectedTableName);
+    assert.ok(table);
+    assertThat(table.columns.length, greaterThan(0));
+    assertThat(table.columns.length, InformationSchemaConfigurator.INFORMATION_SCHEMA_COLUMNS.get(expectedTableName)?.size);
   }
 
-  it('register_shouldRegisterProjectDataSetAndTable', async () => {
+  it('register should register project data set and table', async () => {
     const tableDefinition = new TableDefinition([PROJECT_ID, DATA_SET, TABLE]);
     tableDefinition.schema = ONE_TABLE;
     await shouldRegisterOneTable([tableDefinition], TABLE, [COLUMN_NAME], DATA_SET, PROJECT_ID);
   });
 
-  it('register_shouldRegisterDataSetAndTable', async () => {
+  it('register should register data set and table', async () => {
     const tableDefinition = new TableDefinition([DATA_SET, TABLE]);
     tableDefinition.schema = ONE_TABLE;
     await shouldRegisterOneTable([tableDefinition], TABLE, [COLUMN_NAME], DATA_SET);
   });
 
-  it('register_shouldRegisterOnlyTable', async () => {
+  it('register should register only table', async () => {
     const tableName = `${PROJECT_ID}.${DATA_SET}.${TABLE}`;
     const tableDefinition = new TableDefinition([tableName]);
     tableDefinition.schema = ONE_TABLE;
     await shouldRegisterOneTable([tableDefinition], tableName, [COLUMN_NAME]);
   });
+
+  it('register should register table with time partitioning', async () => {
+    const tableDefinition = new TableDefinition([PROJECT_ID, DATA_SET, TABLE]);
+    tableDefinition.schema = ONE_TABLE;
+    tableDefinition.timePartitioning = true;
+    await shouldRegisterOneTable(
+      [tableDefinition],
+      TABLE,
+      [COLUMN_NAME, ZetaSqlWrapper.PARTITION_TIME, ZetaSqlWrapper.PARTITION_DATE],
+      DATA_SET,
+      PROJECT_ID,
+    );
+  });
+
+  it('register should register information schema', async () => {
+    const tableName = 'columns';
+    const tableDefinition = new TableDefinition([PROJECT_ID, DATA_SET, InformationSchemaConfigurator.INFORMATION_SCHEMA, tableName]);
+    await shouldRegisterInformationSchema([tableDefinition], DATA_SET, tableName, PROJECT_ID);
+  });
+
+  it('register should register information schema when only schema specified', async () => {
```

Comment:
Do we need a test when only region specified? (like in e2e/projects/test-fixture/models/referrers.sql)